packages/@react-aria/disclosure/docs/useDisclosure.mdx

```diff
@@ -119,6 +121,8 @@ function Disclosure(props) {
     display: flex;
     align-items: center;
     gap: 8px;
+    outline: none;
```

Comment:
I think we just need a :focus-visible outline here now.

Reply:
oo good catch